Edited dodofile

PhilLange opened the pull request:
- renamed ica_fit.py to equal definition in dodo.py

PhilLange committed:
added ica solution

PhilLange committed:
renamed file in order to equal dodo file

JoseAlanis commented on 03_repair_eeg_artifacts.py:
```diff
@@ -0,0 +1,141 @@
+# (just in case previous step missed any bad components)
+
+# load template file
+template_raw_file = fname.output(subject=subject,
```
we should have a stardard subject as template, i.e., for comparisson with the subject for which the script ist beeing run. One idea would be to "hard code" the template subject, for instance to be subject 11.
See suggestion below.

JoseAlanis commented on 03_repair_eeg_artifacts.py:
```diff
@@ -0,0 +1,141 @@
+template_raw = read_raw_fif(template_raw_file)
+
+# and template ICA
+template_ica_file = fname.output(subject=subject,
```
```suggestion
template_ica_file = fname.output(subject=11,
```

JoseAlanis commented on 03_repair_eeg_artifacts.py:
```diff
@@ -0,0 +1,141 @@
+
+# compute correlations with template
+corrmap(icas=[template_ica_file, ica],
+        template=(0, 0), threshold=0.9, label='blink_up', plot=False)
```
also here:
in `template=(0, 0)` 
the first 0 in the tuple refers to the 0th ica in the icas argument (which would be the template ICA), the second 0 refers to the 0th component of that ica solution (usually a strong component that tracks eye movements for instance).
not saying the `(0, 0)`ist wrong, but definetily double check the numbers and see if you are correlating with meaningful components

JoseAlanis replied:
And check if they match the `label` argument

JoseAlanis commented on 03_repair_eeg_artifacts.py:
```diff
@@ -0,0 +1,141 @@
+corrmap(icas=[template_ica_file, ica],
+        template=(0, 0), threshold=0.9, label='blink_up', plot=False)
+corrmap(icas=[template_ica_file, ica],
+        template=(0, 4), threshold=0.9, label='blink_side', plot=False)
```
same here:
checke the `template=(0, 4)` argument

PhilLange committed:
- added "repair_artefacts" function to dodo.py
- outcommented eog loop in 03_repair_eeg_artifacts.py
- adjusted ica templates for subject 11

PhilLange committed:
- subject 12 as template, cause there´s a clear ica blink artifact

JoseAlanis committed:
changes in ica fitting and pruning

JoseAlanis reviewed (approved):
- I changed the config file to fix some errors in the labeling of the event codes.
- also added some changes to the fitting of the ICA and selection of bad components.

All in all keeps the logic of the script you added in the last commit.

JoseAlanis commented:
Go ahead and merge if everything ok
